zproject/dev_urls.py

```diff
@@ -18,4 +18,6 @@
     urls += [
         url(r'^user_avatars/(?P<path>.*)$', serve,
             {'document_root': os.path.join(settings.LOCAL_UPLOADS_DIR, "avatars")}),
+        url(r'^realms/(?P<path>.*)$', serve,
+            {'document_root': os.path.join(settings.LOCAL_UPLOADS_DIR, "realms")}),
```

Comment:
This isn't needed if we keep the local uploads directory structure consistent with the S3 structure.